src/wallet/rpcwallet.cpp

```diff
@@ -2350,6 +2350,79 @@ static RPCHelpMan settxfee()
     };
 }
 
+static RPCHelpMan setfeerate()
+{
+    return RPCHelpMan{
+        "setfeerate",
+        "\nSet the transaction fee rate in " + CURRENCY_ATOM + "/vB for this wallet.\n"
+        "Overrides the global -paytxfee configuration option.\n"
+        "Can be deactivated by passing 0 as the fee rate, in which case automatic fee selection will be used by default.\n",
```

Comment:
c6e9bfd1bd9d6a65673fbe1c278a7c7ada35273d

Worth noting that the new settings is not persisted?

Reply:
Thanks, added to settxfee and setfeerate helps